src/components/tasks/TaskList/TaskList.tsx

```diff
@@ -44,14 +43,15 @@ export default function TaskList({
         hasLimit,
         tasksLimit,
     });
+    const [updateTaskDetailsById] = useUpdateTaskDetailsByIdMutation();
+
     function onSeeMoreTasksHandler() {
         setTasksLimit((prevLimit) => prevLimit + ADD_MORE_TASKS_LIMIT);
     }
     async function onContentChangeHandler(id: string, cardDetails: any) {
-        if (!isEditable || !updateCardContent) return;
-        updateCardContent(id, cardDetails);
+        if (!isEditable || !updateTaskDetailsById) return;
```

Comment:
please check is  `!updateTaskDetailsById` required here

Reply:
It is not required, I will remove it.